tests/zfs-tests/tests/functional/delegate/setup.ksh

```diff
@@ -68,7 +69,9 @@ if [ $? -ne 0 ]; then
 fi
 
 DISK=${DISKS%% *}
-default_volume_setup $DISK
-log_must chmod 777 $TESTDIR
 
-log_pass
+if is_linux; then
+	log_must set_tunable64 zfs_admin_snapshot 1
+fi
+
+default_volume_setup $DISK
```

Comment:
`default_volume_setup` will execute `log_pass`, remove unreachable code. Same in `cleanup`.